api/mutualfunds/tests.py

```
@@ -1,7 +1,9 @@
 from django.test import TestCase
+from tablib import Dataset
 
 from mutualfunds.importers.cas import import_cas
-from mutualfunds.models import Portfolio, Folio
+from mutualfunds.importers.master import FundSchemeResource
+from mutualfunds.models import Portfolio, Folio, FundScheme, Transaction, AMC
```

Ruff (0.8.2)

`mutualfunds.models.Transaction` imported but unused

Remove unused import

(F401)